addons/logistics_uavbattery/CfgVehicles.hpp

```diff
@@ -11,6 +11,7 @@ class CfgVehicles {
         };
     };
     class UAV_01_base_F: Helicopter_Base_F {
+        fuelCapacity = 19; // Around 30 minutes hovering
```

Comment:
Is there a formula to calculate these values? Could it be more useful than magic numbers?

Reply:
I couldn't find info on the formula anywhere, it's a BI secret apparently.

We could try asking them, but from my findings it's engine RPM + some other factor I couldn't identify.

Reply:
So fuelConsumptionRate is ignored for carx, tankx, motorcycle?

Reply:
no fuelConsumptionRate is (0.01+rpm*0.02) for cars, as far as i understood it.

Reply:
^ this seems close to what I observed.

Also I don't see anything relating to UAV RPMs in configs, so i'm not sure what's up with that.

Reply:
fuelConsumptionRate is a config entry. It cannot be a formula like that.

Reply:
Exactly. fuelConsumptionRate is a float.
I'm too lazy to check again now. But I think what I wrote about Car==Carx

Reply:
Anything to be done here or good to merge?

Reply:
Good to merge. We might be able to calculate more exact numbers so that they last 30.0 minutes instead of 29.8 minutes or so.. Nobody cares about that.

Reply:
I thought these numbers was calculated somehow and not just guessed. And then formula here instead magic number would be better for future.

Reply:
They were calculated
>Fuel capacity was calculated by giving each drone 1 fuel capacity and making them hover for a minute then multiply the result by the desired battery life.

https://github.com/acemod/ACE3/pull/6647#issue-225022928